sskittenapi/catapi/catapi/dto.py

```diff
@@ -89,13 +89,21 @@ class Cat(BaseModel):
     mtime: datetime
 
 
+class CatUrl(BaseModel):
```

Comment:
[question] As discussed before, what is a `CatUrl`?

Reply:
Just a cat with a URL field
But now this dto is only used by cat_domain.

Reply:
Removed this dto.CatUrl